components/alert.css

```diff
@@ -0,0 +1,36 @@
+.alert-container {
+  box-shadow: 0 0 1px rgb(0 0 0 / 10%), 0 2px 6px rgb(0 0 0 / 10%);
+  padding: 10px;
+  max-width: 30rem;
+}
+
+.alert {
+  padding: 1rem;
+  margin-bottom: 1rem;
+  border: 1px solid transparent;
+  border-radius: 5px;
+}
+.alert-primary {
+  color: var(--primary);
+  background-color: #cfe2ff;
```

Comment:
Suggestion to use root variables instead, can be defined and stored in their own CSS file